command/rubycommand/rubycommand.go

```diff
@@ -213,3 +215,56 @@ func IsGemInstalled(gem, version string) (bool, error) {
 
 	return findGemInList(out, gem, version)
 }
+
+// IsSelectedRbenvRubyInstalled checks if the selected ruby version is installed via rbenv.
+// Ruby version is set by
+// 1. The RBENV_VERSION environment variable
+// 2. The first .ruby-version file found by searching the directory of the script you are executing and each of its
+// parent directories until reaching the root of your filesystem.
+// 3.The first .ruby-version file found by searching the current working directory and each of its parent directories
+// until reaching the root of your filesystem.
+// 4. The global ~/.rbenv/version file. You can modify this file using the rbenv global command.
+func IsSelectedRbenvRubyInstalled(workdir string) (bool, string, error) {
+	absWorkdir, err := pathutil.AbsPath(workdir)
+	if err != nil {
+		return false, "", fmt.Errorf("failed to get absolute path for ( %s ), error: %s", workdir, err)
+	}
+
+	cmd := command.New("rbenv", "version").SetDir(absWorkdir)
+	log.Debugf("$ %s", cmd.PrintableCommandArgs())
+
+	log.Debugf("Ruby version:")
+	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		return false, "", fmt.Errorf("failed to check installed ruby version, error: %s", err)
+	}
+	log.Debugf(out)
+
+	//
+	// Not installed
+	reg, err := regexp.Compile("rbenv: version \x60.*' is not installed")
```

Comment:
this \x60 is very misleading this way, please comment on this

Reply:
Fixed